.github/workflows/win-cpu-arm64-build.yml

```diff
@@ -14,12 +14,28 @@ env:
   ort_dir: "onnxruntime-win-arm64-1.16.3"
   ort_zip: "$(ort_dir).zip"
   ort_url: "https://github.com/microsoft/onnxruntime/releases/download/v1.16.3/$(ort_zip)"
+  VCPKG_FORCE_SYSTEM_BINARIES: "1"
 
 jobs:
   job:
     runs-on: ["self-hosted", "1ES.Pool=onnxruntime-genai-Win11-ARM-CPU"]
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout OnnxRuntime GenAI repo
+        uses: actions/checkout@v4
+      - name: Download latest Git
+        run: |
+          Invoke-WebRequest https://github.com/dennisameling/git/releases/download/v2.41.0.windows.1/PortableGit-2.41.0-arm64.7z.exe -OutFile ${{ github.workspace }}/git.exe
```

Comment:
Are you saying the default git tool's version is too low?
Could you put the git tools in the VM image instead? And we'd better to get the tool from an official place, like https://git-scm.com/download/win. Otherwise we will get risk of leaking our Github token, which may result a massive data leak. 
Windows ARM64 should be able to run x86 binaries. If it doesn't, try to build git from source.

Reply:
default git isn't installed at all. I need to rebuild the image and see if it works.